climakitae/explore/warming.py

```diff
@@ -123,13 +121,12 @@ def calculate(self):
         for level in tqdm(
             self.wl_params.warming_levels, desc="Computing each warming level"
         ):
-            # Assign warming slices to dask computation graph
-            warm_slice = load(
-                self.find_warming_slice(level, self.gwl_times), progress_bar=True
-            )
-            # Dropping simulations that only have NaNs
-            warm_slice = warm_slice.dropna(dim="all_sims", how="all")
```

Comment:
Removed since this causes the warming level data to be loaded into memory. This is being addressed in https://github.com/cal-adapt/climakitae/pull/402.